yb-voyager/src/utils/sqlname/sqlname.go

```diff
@@ -239,7 +240,11 @@ func IsAllUppercase(s string) bool {
 
 func IsAllLowercase(s string) bool {
 	for _, c := range s {
-		if c >= 'A' && c <= 'Z' {
+		if c == '_' {
+			continue
+		}
+		if (c >= 'A' && c <= 'Z') ||
+			!(unicode.IsLetter(rune(c)) || unicode.IsDigit(rune(c))) { // check for special chars
```

Comment:
Looks like unicode pkg also has an IsUpper? See if we can directly use that? https://stackoverflow.com/a/59293875

Another (non-efficient) solution I like is this :) https://stackoverflow.com/a/59293612